apps/studio/src/common/platform_info.ts

```diff
@@ -94,13 +112,19 @@ if (isRenderer()) {
     testMode,
     appDbPath: join(userDirectory, isDevEnv ? 'app-dev.db' : 'app.db'),
     updatesDisabled,
-    appVersion: testMode ? 'test-mode' : e?.app.getVersion() ?? p.env.version,
+    appVersion,
+    parsedAppVersion,
     // cloudUrl: isDevEnv ? 'https://staging.beekeeperstudio.io' : 'https://app.beekeeperstudio.io',
     cloudUrl: 'https://app.beekeeperstudio.io',
     locale,
+    isUltimate: false,
+    isCommunity: true,
 
     // cloudUrl: isDevEnv ? 'http://localhost:3000' : 'https://app.beekeeperstudio.io'
   }
+
+  updateLicenseStatus()
```

Comment:
🐞🐞 This is not ok -- we can't have files run code on import like this -- firstly it doesn't work (crashes the app), but also we need all code like this behind an initializer of some sort.

Instead of `platformInfo.isUltimate`, I suggest something like:
```
const license = await LicenseKey.findBest()
license.isUltimate
```